roles/core/tasks/install.yml

```diff
@@ -39,9 +39,34 @@
     dest: "/tmp/sdcore-5g-values.yaml"
   when: inventory_hostname in groups['master_nodes']
 
+- set_fact:
+    chart_ref_sd_core: "{{ core.helm.chart_ref | regex_replace('^(.*)/.*$', '\\1')  }}"
+  when: inventory_hostname in groups['master_nodes'] and core.helm.local_charts
+
+- debug:
+    var: "chart_ref_sd_core"
+  when: inventory_hostname in groups['master_nodes'] and core.helm.local_charts 
```

Comment:
Why do you need to keep this task?

Reply:
It prints the new path, in case we face any error in the future